docs/includes/_access_token_create_public_client_pkce.mdx

```diff
@@ -1,105 +1,105 @@
+In this flow, you call the app's '/authorize' endpoint, initiating a browser based flow where the user provides their credentials in exchange for a code. In a second API call to the '/token' endpoint, you then exchange the code for an access token.
 
-In this flow, you call the app's '/authorize' endpoint, initiating a browser based flow where the user provides their credentials in exchange for a code. In a second API call to the '/token' endpoint, you then exchange the code for an access token.  
+Note that you will need an identity configured with the [ability to authorize the scopes](/docs/next/api-scopes#scopes-authorized-for-the-identity-approving-the-authorization-request) your app is requesting.
 
-Note that you will need an identity configured with the [ability to authorize the scopes](/docs/next/api-scopes#scopes-authorized-for-the-identity-approving-the-authorization-request) your app is requesting. 
+1. Create an [app](/docs/next/add-an-application) with the following properties:
 
-1. Create an [app](/docs/next/add-an-application) with the following properties:  
+  | Parameter | Value |
+  | --- | --- |
+  | Protocol | OIDC |
+  | Client Type | Public |
+  | PKCE | S256 |
+  | Token Endpoint Auth Method | None |
+  | Grant Type | Authorization Code |
+  | Configuration Type (on Authenticator Config tab) | Hosted Web |
 
-  | Parameter | Value|
-  |---|---|  
-  |Protocol|OIDC|  
-  |Client Type|Public|
-  |PKCE|S256|
-  |Token Endpoint Auth Method|None|
-  |Grant Type|Authorization Code|
-  |Configuration Type (on Authenticator Config tab)|Hosted Web| 
+  Fill in a **Display Name** and at least one **Redirect URI**, then click **Submit** to save the app.
```

Comment:
Can we follow what we do in the Next.js guide where we list Display Name in the parameters and add description and why's to these tables?